icu4j/main/core/src/main/java/com/ibm/icu/number/LocalizedNumberFormatter.java

```diff
@@ -121,6 +121,16 @@ public Format toFormat() {
         return new LocalizedNumberFormatterAsFormat(this, resolve().loc);
     }
 
+    /**
+     * Disassociate the locale from this formatter.
+     *
+     * @return The fluent chain.
+     * @draft ICU 74
+     */
+    public UnlocalizedNumberFormatter resetLocale() {
```

Comment:
I probably missed the discussion, but why `resetLocale()` in Java and `withoutLocale()` in C++?

Reply:
I think I simply forgot to change Java! That is why we have code reviews...